### PR TITLE
[2.0.x] Italian language fix

### PR DESCRIPTION
**Pull request by GMagician**
Added missing messages and fixed some other

### CODE REVIEW

**Comment by GMagician** [comment]
N.B it seems that `MSG_FILAMENT_CHANGE_HEAT_1` English define is missing when `LCD_HEIGHT < 4`

File: Marlin/src/lcd/language/language_it.h [hunk 1]
```diff
@@ -300,26 +309,26 @@
...
 #define MSG_DELTA_HEIGHT                    _UxGT("Altezza")
 #define MSG_DELTA_RADIUS                    _UxGT("Raggio")
 #define MSG_INFO_MENU                       _UxGT("Riguardo stampante")
 #define MSG_INFO_PRINTER_MENU               _UxGT("Info. stampante")
 #define MSG_3POINT_LEVELING                 _UxGT("Livel. a 3 punti")
 #define MSG_LINEAR_LEVELING                 _UxGT("Livel. Lineare")
-#define MSG_BILINEAR_LEVELING               _UxGT("Livel. Biliniare")
-#define MSG_UBL_LEVELING                    _UxGT("Livel. UBL")
+#define MSG_BILINEAR_LEVELING               _UxGT("Livel. Bilineare")
+#define MSG_UBL_LEVELING                    _UxGT("Livel. piatto unificato")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_it.h]: This is too long:
```
12345678901234567890
Livel. piatto unific|ato
```

**Comment by thinkyhead** [comment]
Thanks for the note! Added:
```cpp
  #ifndef MSG_FILAMENT_CHANGE_HEAT_1
    #define MSG_FILAMENT_CHANGE_HEAT_1          _UxGT("Click to heat")
  #endif
```

File: Marlin/src/lcd/language/language_it.h [hunk 2]
```diff
@@ -98,18 +103,18 @@
...
+#define MSG_UBL_DONE_EDITING_MESH           _UxGT("Modif.Mesh fatta")
+#define MSG_UBL_BUILD_CUSTOM_MESH           _UxGT("Crea Mesh Custom")
 #define MSG_UBL_BUILD_MESH_MENU             _UxGT("Crea Mesh")
-#define MSG_UBL_BUILD_PLA_MESH              _UxGT("Crea PLA Mesh")
-#define MSG_UBL_BUILD_ABS_MESH              _UxGT("Crea ABS Mesh")
-#define MSG_UBL_BUILD_COLD_MESH             _UxGT("Crea Cold Mesh")
+#define MSG_UBL_BUILD_PLA_MESH              _UxGT("Crea Mesh PLA")
+#define MSG_UBL_BUILD_ABS_MESH              _UxGT("Crea Mesh ABS")
+#define MSG_UBL_BUILD_COLD_MESH             _UxGT("Crea Mesh a freddo")
 #define MSG_UBL_MESH_HEIGHT_ADJUST          _UxGT("Aggiusta Altezza Mesh")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_it.h]: This is too long also:
```
12345678901234567890
Aggiusta Altezza Mes|h
```

File: Marlin/src/lcd/language/language_it.h [hunk 3]
```diff
@@ -286,8 +294,9 @@
 #define MSG_ENDSTOP_ABORT                   _UxGT("Finecorsa abort")
 #define MSG_HEATING_FAILED_LCD              _UxGT("Riscald. Fallito")
 #define MSG_HEATING_FAILED_LCD_BED          _UxGT("Risc. piatto fallito")
-#define MSG_ERR_REDUNDANT_TEMP              _UxGT("Err: TEMP RIDONDANTI")
+#define MSG_ERR_REDUNDANT_TEMP              _UxGT("Err: TEMP RIDONDANTE")
 #define MSG_THERMAL_RUNAWAY                 _UxGT("TEMP FUORI CONTROLLO")
+#define MSG_THERMAL_RUNAWAY_BED             _UxGT("TEMP PIATTO FUORI CTRL.")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_it.h]:
```
12345678901234567890
TEMP PIATTO FUORI CT|RL.
```

File: Marlin/src/lcd/language/language_it.h [hunk 4]
```diff
@@ -286,8 +294,9 @@
...
 #define MSG_ERR_MAXTEMP                     _UxGT("Err: TEMP MASSIMA")
 #define MSG_ERR_MINTEMP                     _UxGT("Err: TEMP MINIMA")
 #define MSG_ERR_MAXTEMP_BED                 _UxGT("Err: TEMP MASSIMA PIATTO")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_it.h]:
```
12345678901234567890
Err: TEMP MASSIMA PI|ATTO
```

File: Marlin/src/lcd/language/language_it.h [hunk 5]
```diff
@@ -79,16 +81,19 @@
...
+#define MSG_IDEX_MENU                       _UxGT("Modo IDEX")
+#define MSG_IDEX_MODE_AUTOPARK              _UxGT("Auto-Park")
+#define MSG_IDEX_MODE_DUPLICATE             _UxGT("Duplicazione")
+#define MSG_IDEX_MODE_FULL_CTRL             _UxGT("Pieno controllo")
 #define MSG_UBL_MANUAL_MESH                 _UxGT("Mesh Manuale")
 #define MSG_UBL_BC_INSERT                   _UxGT("Metti spessore & misura")
 #define MSG_UBL_BC_INSERT2                  _UxGT("Misura")
 #define MSG_UBL_BC_REMOVE                   _UxGT("Rimuovi e misura piatto")
-#define MSG_UBL_MOVING_TO_NEXT              _UxGT("Spostamento sucessivo")
+#define MSG_UBL_MOVING_TO_NEXT              _UxGT("Spostamento successivo")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_it.h]:
```
12345678901234567890
Metti spessore & mis|ura 
Rimuovi e misura pia|tto
Spostamento successi|vo
```

File: Marlin/src/lcd/language/language_it.h [hunk 6]
```diff
@@ -79,16 +81,19 @@
 #define MSG_NEXT_CORNER                     _UxGT("Prossimo spigolo")
 #define MSG_EDITING_STOPPED                 _UxGT("Modifica Mesh Fermata")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_it.h]:
```
12345678901234567890
Modifica Mesh Fermat|a
```

**Comment by thinkyhead** [comment]
And a few other strings need new shorter translations, also.

**Comment by GMagician** [comment]
@thinkyhead I reduced all messages above 20 chars. I Hope not to miss some

File: Marlin/src/lcd/language/language_it.h [hunk 7]
```diff
@@ -77,18 +79,21 @@
...
+#define MSG_IDEX_MODE_AUTOPARK              _UxGT("Auto-Park")
+#define MSG_IDEX_MODE_DUPLICATE             _UxGT("Duplicazione")
+#define MSG_IDEX_MODE_FULL_CTRL             _UxGT("Pieno controllo")
 #define MSG_UBL_MANUAL_MESH                 _UxGT("Mesh Manuale")
-#define MSG_UBL_BC_INSERT                   _UxGT("Metti spessore & misura")
+#define MSG_UBL_BC_INSERT                   _UxGT("Metti spes. e misura")
 #define MSG_UBL_BC_INSERT2                  _UxGT("Misura")
-#define MSG_UBL_BC_REMOVE                   _UxGT("Rimuovi e misura piatto")
-#define MSG_UBL_MOVING_TO_NEXT              _UxGT("Spostamento sucessivo")
+#define MSG_UBL_BC_REMOVE                   _UxGT("Rimuovi e mis. piatto")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_it.h]: Just one left!
```
12345678901234567890
Rimuovi e mis. piatt|o
```

**Comment by thinkyhead** [comment]
Grazi!

**Comment by GMagician** [comment]
> Grazi!

Thanks to you for all the time you dedicate to this project

**Pull request merged**